### PR TITLE
feature/#2948-modal-windows-for-ubs-client-profile

### PR DESCRIPTION
**Pull request by rostikdoroshenko**
- created modal window 'change password';
- created modal window 'delete profile';
- added base styles.
change-password
delete-profile

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts [hunk 1]
```diff
@@ -43,6 +48,20 @@ export class UbsClientProfilePageComponent implements OnInit {
     });
   }
 
+  public openDeleteProfileDialog() {
+    const dialogRef = this.dialog.open(UbsProfileDeletePopUpComponent, {
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed().subscribe();
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts:55]: why do you need to subscribe here ?

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts [hunk 2]
```diff
@@ -43,6 +48,20 @@ export class UbsClientProfilePageComponent implements OnInit {
...
+  }
+
+  public openChangePasswordDialog() {
+    const dialogRef = this.dialog.open(UbsProfileChangePasswordPopUpComponent, {
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed().subscribe();
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts:62]: why do you need to subscribe here ?

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<div class="dialog-container">
+  <h2 mat-dialog-title>{{ 'Зміна паролю' }}</h2>
+  <form [formGroup]="formConfig">
+    <div class="input">
+      <mat-form-field [style.width.px]="200" class="forms-inputs">
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.html:5]: move all inline styles to classes

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<div class="dialog-container">
+  <h2 mat-dialog-title>{{ 'Зміна паролю' }}</h2>
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.html:2]: are you sure you need to have this syntax for simple text ? can this be  <h2 mat-dialog-title>Зміна паролю</h2>

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    this.initForm();
+  }
+
+  onSubmit() {
+    if (!this.formConfig.errors) {
+      this.changePassword();
+    }
+  }
+
+  changePassword() {}
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.ts:26]: why do you need this ?

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-delete-pop-up/ubs-profile-delete-pop-up.component.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-ubs-profile-delete-pop-up',
+  templateUrl: './ubs-profile-delete-pop-up.component.html',
+  styleUrls: ['./ubs-profile-delete-pop-up.component.scss']
+})
+export class UbsProfileDeletePopUpComponent implements OnInit {
+  ngOnInit() {}
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-profile-delete-pop-up/ubs-profile-delete-pop-up.component.ts:9]: remove an empty ngOnInit

**Review by Anastasiia-Hevchuk — APPROVED** [review]

**Pull request closed without merging**